src/EPR.Calculator.Frontend/Views/Shared/_ValidationErrorsListPartial.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@model List<CreateLapcapDataErrorDto>;
```

Comment:
I think this partial view is shared with default parameters. I think we do not need a partial view in this case as the model is different. Could you please check?